chapter_optimization/momentum_vn.md

```diff
@@ -260,7 +260,7 @@ Let us look at what leaky averaging actually does to the updates.
 ### Effective Sample Weight
 -->
 
-### *dịch tiêu đề phía trên*
+### Trọng số mẫu hiệu quả
```

Comment:
```suggestion
### Trọng số mẫu thực sự
```
`effective` ở đây mang nghĩa `actual == thực sự/ thực tế` ấy, giống cụm `the effective ruler == người cai trị thực sự`